go/vt/vttablet/tabletserver/tx_engine.go

```diff
@@ -428,21 +458,21 @@ outer:
 	return allErr.Error()
 }
 
-// shutdownTransactions rolls back all open transactions
-// including the prepared ones.
-// This is used for transitioning from a primary to a non-primary
-// serving type.
+// shutdownTransactions rolls back all open transactions that are idol.
```

Comment:
idol?

Reply:
idol transactions are the ones that are not running any command right now. Like the user ran a few DMLs, but currently isn't running anything (commit or DML)

Reply:
I believe @systay  is pointing to the type `idol` -> `idle`